app/policies/dish_policy.rb

```diff
@@ -0,0 +1,18 @@
+class DishPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def create?
+    user.present?
```

Comment:
for now just true. Because authorisation system task is not for policy, but for devise

Reply:
If user.admin? only